internal/lexer/gen/golang/lexer.go

```diff
@@ -180,39 +180,39 @@ func (this *Lexer) Scan() (tok *token.Token) {
 
 			switch rune1 {
 			case '\n':
-				this.line++
-				this.column = 1
+				l.line++
+				l.column = 1
 			case '\r':
-				this.column = 1
+				l.column = 1
 			case '\t':
-				this.column += 4
+				l.column += 4
 			default:
-				this.column++
+				l.column++
 			}
 
 			switch {
 			case ActTab[state].Accept != -1:
 				tok.Type = ActTab[state].Accept
 				// fmt.Printf("\t Accept(%s), %s(%d)\n", string(act), token.TokMap.Id(tok), tok)
```

Comment:
I'd say to just throw in a commit deleting the 2 dead lines [R196](https://github.com/goccmack/gocc/pull/52/files#diff-7159126bf8e8e06b2c7cc66df336ddd5R196) and [R199](https://github.com/goccmack/gocc/pull/52/files#diff-7159126bf8e8e06b2c7cc66df336ddd5R199). But sure.